ship.config.js

```diff
@@ -14,7 +14,6 @@ module.exports = {
     // update metadata.yaml by executing external script instead of inline because
     // the esm module loader being used by Ship.js is incompatible with the YAML package
     exec(`node scripts/update-metadata.js ${tag}`);
-    exec(`git add metadata.yaml`);
-    exec(`git commit -m "chore: update metadata.yml"`);
+    return `git commit -am "chore: update metadata.yml"`;
```

Comment:
We don't need to specify the `metadata.yaml` file?

Reply:
The `-a` (or `--all`) part of the git commit flag will automatically stage any modified or deleted files.

We could also do `git commit -m "chore: update metadata.yml" metadata.yaml`, but I figured since no other files would be modified by the script then it'd be relatively safe to just stage any modified files.

Reply:
Yeah okay. I figured there's always a chance that some rando file will get committed, but it's probably unlikely 🤷‍♀️